pages/multiplexed-samples/[uuid].js

```diff
@@ -0,0 +1,199 @@
+            <>
+              <DataAreaTitle>Multiplexed Samples</DataAreaTitle>
+              <SampleTable samples={multiplexedSample.multiplexed_samples} />
+            </>
+          )}
+          {multiplexedSample.file_sets?.length > 0 && (
+            <>
+              <DataAreaTitle>File Sets</DataAreaTitle>
+              <FileSetTable fileSets={multiplexedSample.file_sets} />
+            </>
+          )}
+          {multiplexedSample.modifications?.length > 0 && (
+            <>
+              <DataAreaTitle>Modifications</DataAreaTitle>
+              <ModificationsTable
+                modifications={multiplexedSample.modifications}
+              />
+            </>
+          )}
+          {biomarkers.length > 0 && (
+            <>
+              <DataAreaTitle>Biomarkers</DataAreaTitle>
+              <BiomarkerTable biomarkers={biomarkers} />
+            </>
+          )}
+          {treatments.length > 0 && (
+            <>
+              <DataAreaTitle>Treatments</DataAreaTitle>
+              <TreatmentTable treatments={treatments} />
+            </>
+          )}
+          {documents.length > 0 && (
+            <>
+              <DataAreaTitle>Documents</DataAreaTitle>
+              <DocumentTable documents={documents} />
+            </>
+          )}
+          <Attribution attribution={attribution} />
+        </JsonDisplay>
+      </EditableItem>
+    </>
+  );
+}
+
+MultiplexedSample.propTypes = {
+  // MultiplexedSample-cell sample to display
+  multiplexedSample: PropTypes.object.isRequired,
+  // Documents associated with the sample
+  documents: PropTypes.arrayOf(PropTypes.object).isRequired,
+  // Donors associated with the sample
+  sources: PropTypes.arrayOf(PropTypes.object),
+  // Treatments associated with the sample
+  treatments: PropTypes.arrayOf(PropTypes.object).isRequired,
+  // Biomarkers of the sample
+  biomarkers: PropTypes.arrayOf(PropTypes.object).isRequired,
+  // Attribution for this sample
+  attribution: PropTypes.object,
+  // Is the format JSON?
+  isJson: PropTypes.bool.isRequired,
+};
+
+export async function getServerSideProps({ params, req, query }) {
+  const isJson = isJsonFormat(query);
+  const request = new FetchRequest({ cookie: req.headers.cookie });
+  const multiplexedSample = await request.getObject(
+    `/multiplexed-samples/${params.uuid}/`
+  );
+  if (FetchRequest.isResponseSuccess(multiplexedSample)) {
+    const documents = multiplexedSample.documents
+      ? await requestDocuments(multiplexedSample.documents, request)
+      : [];
+    const donors = multiplexedSample.donors
+      ? await requestDonors(multiplexedSample.donors, request)
+      : [];
+    let sources = [];
+    if (multiplexedSample.sources?.length > 0) {
+      const sourcePaths = multiplexedSample.sources.map(
+        (source) => source["@id"]
+      );
+      sources = await request.getMultipleObjects(sourcePaths, null, {
+        filterErrors: true,
+      });
```

Comment:
@khzlin Since I was too lazy to add a comment to InVitroSystem, could you add a short comment here explaining why you use `getMultipleObjects` and that it shouldn’t use `getMultipleObjectsBulk`?

Reply:
That way someone down the line doesn’t swap it out and waste time reverting it back.